iocBoot/iocBPMRFFEApp/run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -ue
+
+# Load prefix mappings
+. slot-mapping.env
+
+# Define which env var to use as prefix
+area_prefix_var=CRATE_${CRATE_NUMBER}_BPM_${BPM_NUMBER}_PV_AREA_PREFIX
+device_prefix_var=CRATE_${CRATE_NUMBER}_BPM_${BPM_NUMBER}_PV_DEVICE_PREFIX
+
+# Indirectly expand prefixes
+export EPICS_PV_AREA_PREFIX=${!area_prefix_var}
+export EPICS_PV_DEVICE_PREFIX=${!device_prefix_var}
+
+# Use base address and shift it by BPM slot number
+IFS="." read -a ip <<< $RFFE_BASE_IP_ADDRESS
+export RFFE_IP_ADDRESS=${ip[0]}.${ip[1]}.${ip[2]}.$(( ${ip[3]} + $BPM_NUMBER ))
+
+socket_path=/var/run/procServ/procserv.sock
+mkdir -p $(dirname $socket_path)
+mkdir -p /opt/autosave/rffe
+
+# Used by procServ and autosave
+export DEVICE_NAME="${EPICS_PV_AREA_PREFIX}${EPICS_PV_DEVICE_PREFIX::-1}"
+
+procServ -f -n $DEVICE_NAME -i ^C^D -L - unix:$socket_path ./st.cmd
```

Comment:
Got it. Forgot that it might be modified if exported. I did so to reuse the definition later. I'll fix it. Thanks for the notice.

Reply:
> Forgot that it might be modified if exported.

That's not the only reason; it's also easier on someone reading the `.cmd` files to have as few variables around.

Reply:
Good point. I agree.

Reply:
I updated it. I really don't like having to use characters (`:`) that can break the POSIX Fully Portable Filenames (i.e. only using `A–Z a–z 0–9 . _ -`).

But for the sake of following the standard, I updated it to use `$(P)$(R)` (including the pending semicolon).